src/config/env.ts

```diff
@@ -26,10 +26,9 @@ if (env.error && nodeEnv !== EnvironmentEnum.production) {
 	throw env.error;
 }
 
-const getProxyFromEnvironment = (): string => {
-	const proxyHost = process.env.EXTERNAL_ONLY_PROXY_HOST;
-	const proxyPort = process.env.EXTERNAL_ONLY_PROXY_PORT;
-	return proxyHost && proxyPort ? `http://${proxyHost}:${proxyPort}` : undefined;
+const getProxyFromEnvironment = (): string | undefined => {
+	const proxy = process.env.WHITELIST_PROXY;
```

Comment:
Can you give some details on how it works?

1. This env var is set by Micros, right? Do we need to configure anything in the Micros descriptor?

2. Are we using this proxy for all outbound calls, correct? Would it change the IPs for calls we make to GitHub? If yes, would it require any changes in the IPs whitelist on the GitHub side?